backend/src/main/java/reviewme/review/service/mapper/ReviewMapper.java

```diff
@@ -62,20 +62,10 @@ private List<Answer> getAnswersByQuestionType(ReviewRegisterRequest request) {
 
     private Answer mapRequestToAnswer(Map<Long, Question> questions, ReviewAnswerRequest answerRequest) {
         Question question = questions.get(answerRequest.questionId());
-
         if (question == null) {
             throw new SubmittedQuestionNotFoundException(answerRequest.questionId());
         }
 
-        // TODO: 아래 코드를 삭제해야 한다
-        if (question.isSelectable() && answerRequest.selectedOptionIds() != null && answerRequest.selectedOptionIds().isEmpty()) {
-            return null;
-        }
-        if (!question.isSelectable() && answerRequest.text() != null && answerRequest.text().isEmpty()) {
-            return null;
-        }
-        // END
-
         AnswerMapper answerMapper = answerMapperFactory.getAnswerMapper(question.getQuestionType());
         return answerMapper.mapToAnswer(answerRequest);
```

Comment:
```suggestion
        QuestionType questionType = question.getQuestionType();
        return answerMapperFactory.getAnswerMapper(questionType)
                .mapToAnswer(answerRequest);
```
하나의 제안입니다 무시해도 괜찮아요 (AnswerMapper를 변수로 굳이 둘 필요가 없다?)

Reply:
좋아보여서 반영합니다😄